internal/cmd/group_utils.go

```diff
@@ -0,0 +1,36 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/tursodatabase/turso-cli/internal"
+	"github.com/tursodatabase/turso-cli/internal/turso"
+)
+
+func ensureGroupAwake(client *turso.Client, groupName string) (bool, error) {
```

Comment:
I'm not a fan of this approach because:
- We add an extra call on all invocations to handle an edge case that happens once in a while
- We have to keep track of all ops that are affected by archiving and remember to add this logic to them (e.g. rotating creds, and deleting dbs also doesn't work when the group is archived)

The "better" way would be to be able to detect archived errors from API responses and handle those properly. 

Not a blocker though, feel free to move forward if you don't have the time to address this.

Reply:
I agree this would be better.